### PR TITLE
Menu tidy up

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

File: docs/themerc [hunk 1]
```diff
@@ -41,3 +45,4 @@ osd.bg.color: #dddda6
 osd.border.color: #000000
 osd.border.width: 1
 osd.label.text.color: #000000
+
```
Comment (Consolatis) [inline docs/themerc:48]: New empty line. intended?

**Comment by Consolatis** [comment]
Commit message misses the `padding` part in `menu.items.{x,y}`.
Suggest: docs/themerc: add new menu settings because Consolatis didn't pay attention.

Nit: feels a bit strange to have `menu.overlap` on top and then a block for the other menu entries below.

Other than that: LGTM

File: docs/themerc (hunk 1 above)
Comment (johanmalm) [reply docs/themerc]: Not intended. Removed and force-pushed.

**Comment by johanmalm** [comment]
> Commit message misses the `padding` part in `menu.items.{x,y}`.

Sorted.
 
> Nit: feels a bit strange to have `menu.overlap` on top and then a block for the other menu entries below.

Agree. I've moved `menu.overlap.*` down to the menu section.

**Comment by johanmalm** [comment]
The ordering of the `themerc` was originally in accordance with http://openbox.org/wiki/Help:Themes#Theme_file_structure - but it does seems a bit odd having menu.overlap at the top :smile:

**Pull request merged**